### DISCUSSION TITLE
REST API dla kupującego (zamówienia/zwroty)

### DISCUSSION THREAD

**Discussion by piotrraczynski** [allegro/allegro-api#5285] [Suggestions - rejected]
Chciałbym zaproponować REST API dedykowane dla kupującego w zakresie zwrotów. Kupujący bardzo dużo na pewno doceniliby możliwość uzyskania listy swoich zamówień i zwrotów (w tym terminów) z zewnętrznych aplikacji.

Głównie mam na myśli API funkcje umożliwiające zarządzać zwrotami po stronie kupującego.

**Comment by MartaNowaczyk** [comment]
Po weryfikacji sugestii przez odpowiedni zespół sugestię musze odrzucić, ze względu na przewidywaną niską skalę użycia.
